### PR TITLE
fix #285464: downgrade to Qt 5.9 for NVDA accessibility

### PR DESCRIPTION
**Pull request by shoogle**
Issue: https://musescore.org/en/node/285464

### CODE REVIEW

**Comment by shoogle** [comment]
I ran a [full AppVeyor build on my own fork](https://ci.appveyor.com/project/shoogle/musescore/builds/22918675) for 32 and 64 bit and tested the resulting artifacts. They work as expected.

**Comment by Jojo-Schmitz** [comment]
Workaround for https://github.com/nvaccess/nvda/issues/8604 ?

File: build/appveyor/before_build.bat [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
   :: SET "QTDIR=%cd%\qt\msvc2017_64" & :: uncomment to use our Qt
-  SET "QTDIR=C:\Qt\5.12\msvc2017_64" & :: uncomment to use AppVeyor's Qt
+  SET "QTDIR=C:\Qt\5.9\msvc2017_64" & :: uncomment to use AppVeyor's Qt
   SET "TARGET_PROCESSOR_BITS=64"
   SET "TARGET_PROCESSOR_ARCH=x86_64"
 ) ELSE (
   :: SET "QTURL=https://utils.musescore.org.s3.amazonaws.com/qt5120_msvc2017_32.7z"
   :: SET "QTDIR=%cd%\qt\msvc2017" & :: uncomment to use our Qt
-  SET "QTDIR=C:\Qt\5.12\msvc2017" & :: uncomment to use AppVeyor's Qt
+  SET "QTDIR=C:\Qt\5.9\msvc2015" & :: uncomment to use AppVeyor's Qt
```
Comment (Jojo-Schmitz) [inline build/appveyor/before_build.bat]: This means no WebEngine, that requires MSVC 2017 (and in Qt 5.9 also 64bit)

Comment (shoogle) [reply build/appveyor/before_build.bat]: I downloaded the [32-bit AppVeyor build](https://ci.appveyor.com/project/shoogle/musescore/builds/22918675) and the WebView in the Start Center works fine, as does Save Online. What makes you think it wouldn't work?

**Comment by Jojo-Schmitz** [comment]
The comment in Qt 5.9 when selecting WebEngine claims that it plays together only with MSVC 2017 64bit

**Comment by shoogle** [comment]
Interesting. That comment is only displayed for 5.9.7 and not 5.9.6, etc. so it could be a mistake. On the other hand I see various comments online suggesting that it is true and if 32 bit WebEngine then you need to compile Qt from source, so it is possible that AppVeyor compile Qt themselves rather than bundling the pre-built binaries provided by Qt.

In any case, I checked the binaries in the AppVeyor build and they are all 32 bit, including QWebEngineProcess.exe and all the DLLs.

**Comment by anatoly-os** [comment]
The PR is out of date since 5.12.4 contains the fix which enables the compatibility. Btw, 5.12.4 contains a serious regression, so we will wait for qt 5.12.5.

**Pull request closed without merging**